### DISCUSSION TITLE
Launch stack 1.1.3: Thumbnail images are forbidden

### DISCUSSION THREAD

**Discussion by kirkglpro** [mozilla/hubs#5201] [Hubs Cloud Questions] [answered]
After upgrading to the latest launch stack (1.1.3) the thumbnails of scene's and avatars in the admin panel throw a 403 forbidden error. The rooms however and working fine without any issues. I am using a custom hubs client build.

One thing i noticed is from the avatar and scenes listing on admin the resources are loaded from https://domain/files/uuid which throws a 403

If stack is named **myhubs**
However, the images load from https://sub.domain/files/uuid (https://myhubs.domain.com/files/uuid) if i open the image in a new tab. 

Also, the images in the App Settings page (IMAGES tab) also look good and load from https://myhubs.domain.com/files/uuid

Kindly assist how to fix this.

Avatar\ Scene listing
![image](https://user-images.githubusercontent.com/93887528/156499671-019607b5-ec60-450a-9043-ea509e18a3bf.png)

SSH Log
![image](https://user-images.githubusercontent.com/93887528/156499719-b03f91e0-2f70-40ed-83ff-949acdca9e15.png)

**Comment by pattersonbl2** [comment, marked as answer]
I couldn't reproduce this in my 1.1.3 stack.   Since it's a custom client,  How did you upgrade to 1.1.3?   Also, are you using the latest code in the hubs cloud branch?   There is a single lambda that controls thumbnails that had some code changes.  This would be called speelycaptor.

**Reply by kirkglpro** [reply]
@Pattersonbl2 I upgraded from this tutorial shared earlier [Upgrade launch stack](https://hubs.mozilla.com/docs/hubs-cloud-aws-updating-the-stack.html#-upgrade-your-hubs-cloud-template-to-the-latest).

I am using a custom client and not upgraded to latest code base.

**Reply by pattersonbl2** [reply]
I would look to upgrade the code base because we have made some changes in some repo that cause this issue.

**Reply by kirkglpro** [reply]
@Pattersonbl2 Alright! that fixed the issue. New client update works fine. Thanks for your feedback.

**Reply by kirkglpro** [reply]
@Pattersonbl2 There is one issue though. In the recent master branch update the thumbnail and preview are both in the table.
https://github.com/mozilla/hubs/blob/master/admin/src/react-components/avatars.js
The table is probably duplicating elements and can be removed.
![image](https://user-images.githubusercontent.com/93887528/156905487-045d9013-cdf0-41a6-8d1e-4cd225a13a79.png)

**Reply by pattersonbl2** [reply]
I believe the most updated branch isn't the master but the hub cloud branch.

**Reply by kirkglpro** [reply]
Thanks @Pattersonbl2 I will have that updated on my end
